providers/aws/provider.go

```diff
@@ -302,7 +302,7 @@ func (p *FargateProvider) NodeConditions(ctx context.Context) []corev1.NodeCondi
 			Message:            lastTransitionMessage,
 		},
 		{
-			Type:               corev1.NodeKubeletConfigOk,
```

Comment:
This **NodeKubeletConfigOK** is removed in https://github.com/kubernetes/api/commit/5236860eec85df3505d9c15f0bfc4f7229a888c9#diff-9df476defa691b9e80d8c2cb703eeafb